### DISCUSSION TITLE
Antimalware Win 11: HEUR threat detected

### DISCUSSION THREAD

**Discussion by eugenio-novosolov** [Sidekick-Poe/Sidekick#372] [Q&A]
Hey community!I would love to start using Sidekick, but.. My Anti-malware agent on windows 11 (NordVPN threat protection) removes downloaded package immediately saying` THREAT DETECTED: HEUR/APC`
I realise this kind of issue would not go into work anytime soon, as it does not seem to be a common pain here. So maybe someone had similar issue? NordVPN does not allow whitelist anything, and I don't want to disable it completely.
If somebody understands what kind of work needs to be done to fix that on the codebase side, I could contribute to make it happen.
Let the force be with you, Exiles.

**Comment by domialex** [comment]
There is nothing we can do on our side to prevent third-party programs from blocking Sidekick unfortunately. Maybe you'll have more success with the portable version of Sidekick.
